extensions/amp-iframe/1.0/component.js

```diff
@@ -30,6 +35,42 @@ export function Iframe({
   const isIntersectingRef = useRef(null);
   const containerRef = useRef(null);
 
+  const viewabilityCb = (entries) => {
+    const iframe = iframeRef.current;
+    if (!iframe) {
+      return;
+    }
+    const win = toWin(iframe.ownerDocument.defaultView);
+    postMessage(
+      iframe,
+      MessageType.INTERSECTION,
+      cloneEntryForCrossOrigin(entries[0]),
```

Comment:
This doesn't match the `INTERSECTION` API. We need to send a `{ changes: Entry[] }` object

Reply:
Done.